R/PipeOpTaskSurvClassif.R

```diff
@@ -0,0 +1,71 @@
+PipeOpTaskSurvClassif <- R6Class(
+  "PipeOpTaskSurvClassif",
+  inherit = PipeOpTaskTransformer,
+
+  public = list(
+    initialize = function(id = "trafotask_survclassif") {
+      super$initialize(id = id,
+                       input = data.table::data.table(name = "input", train = "TaskSurv", predict = "TaskSurv"),
+                       output = data.table::data.table(name = c("output", "meta"), train = c("TaskClassif", "*"), predict = c("TaskClassif", "*")))
+    }
+  ),
+
+  private = list(
+    .train = function(task, cut = NULL) {
+      cut = NULL
+      task = task[[1]]
+      data = task
+
+      assert_task(task, task_type = "surv")
+      assert_numeric(cut, null.ok = TRUE, lower = 0)
+
+      time = task$target_names[1]
+      event = task$target_names[2]
+
+      formula = sprintf("Surv(%s, %s) ~ .", time, event)
+
+      # TODO: do without pammtools
+      long_data = pammtools::as_ped(as.formula(formula), data = task$data(), cut = cut)
```

Comment:
@adibender if we will need to import `pammtools` at some point then we might as well leave it as that, otherwise let's write it in base R